js/accordionModel.js

```diff
@@ -0,0 +1,74 @@
+define([
+    'core/js/models/itemsModel'
+], function(ItemsModel) {
+
+    var AccordionModel = ItemsModel.extend({
+
+        defaults: function() {
+            return _.extend({
+                _shouldCollapseItems: true,
+                _toggleSpeed: 200,
+                _activeItems: []
+            }, ItemsModel.prototype.defaults);
+        },
+
+        initialize: function() {
+            if (this.get('_shouldCollapseItems') === false) {
+                this.set('_activeItems', []);
+            }
+
+            _.each(this.get('_items'), function(item) {
+                item._isActive = false;
+            });
+
+            ItemsModel.prototype.initialize.apply(this, arguments);
+        },
+
+        checkCompletionStatus: function() {
+            if (this.getVisitedItems().length == this.get('_items').length) {
+                this.setCompletionStatus();
+            }
+        },
+
+        toggleActiveItems: function(itemIndex) {
+            var activeItems = this.get('_activeItems');
```

Comment:
I think instead of having an array for ```_activeItems``` it would be more sensible to have a flag on the item. Much like ```_isVisited``` but ```_isActive```.

This way we can move the ```_activeItems``` behaviour back into the ```ItemsModel``` with the functions ```resetActiveItems()```, ```setItemAsActive(index)```, ```getActiveItems()``` and ```getActiveItemsCount()```.

Admittedly it isn't necessary for ```Hotgraphic``` and ```Narrative``` to have multiple active items at present.